src/test/rating.spec.js

```diff
@@ -0,0 +1,120 @@
+import {
+  generateArticle,
+  ratings,
+  createUser,
+  createArticle
+} from './utils/helpers';
+
+chai.use(chaiHttp);
+chai.should();
+
+const { User, Article, Rating } = db;
+
+describe('PATCH, /api/v1/articles/:id/ratings', () => {
+  beforeEach(async () => {
+    await User.destroy({ truncate: true, cascade: true });
+    await Rating.destroy({ truncate: true, cascade: true });
+    await Article.destroy({ truncate: true, cascade: true });
+  });
+  it('Should return a 400 status code if the token was not set with relevant error message', done => {
+    chai
+      .request(app)
+      .patch('/api/v1/articles/:articleId/ratings')
+      .end((error, response) => {
+        expect(response.status).to.equal(400);
+        expect(response.body.message).to.equal(
+          'Please assign a access token as header'
+        );
+        done();
+      });
+  });
+  it('Should return a relevant error message if the article Id is not valid', async () => {
+    const newUser = await createUser();
+    const token = sign.jwtSigner(newUser);
+
+    const numberOfUsers = await BaseRepository.findAll(User, {});
+    expect(numberOfUsers.length).to.equal(1);
+    const response = await chai
+      .request(app)
+      .patch(`/api/v1/articles/${'foo'}/ratings`)
+      .set('x-access-token', token);
+    response.body.error.should.be.a('string');
+    expect(response.body.error).to.equal('articleId must be a number');
+  });
+  it('Should return an relevant error message if the ratings passed is not a number', async () => {
+    const newUser = await createUser();
+    const token = sign.jwtSigner(newUser);
+
+    const numberOfUsers = await BaseRepository.findAll(User, {});
+    expect(numberOfUsers.length).to.equal(1);
+    const response = await chai
+      .request(app)
+      .patch(`/api/v1/articles/${1}/ratings`)
+      .set('x-access-token', token)
+      .send({ ratings: 'free' });
+    expect(response.status).to.equal(400);
+    response.body.error.should.be.a('string');
+    expect(response.body.error).to.equal('ratings must be a number');
+  });
+  it('Should return a 400 if an author tries ratings his/her article', async () => {
+    const newUser = await createUser();
+    const token = sign.jwtSigner(newUser);
+    const article = await generateArticle({ authorId: newUser.id });
+    const createAnArticle = await createArticle(article);
+
+    const numberOfUsers = await BaseRepository.findAll(User, {});
+    const numberOfArticles = await BaseRepository.findAll(Article, {});
+    expect(numberOfUsers.length).to.equal(1);
+    expect(numberOfArticles.length).to.equal(1);
+    const response = await chai
+      .request(app)
+      .patch(`/api/v1/articles/${createAnArticle.id}/ratings`)
+      .set('x-access-token', token)
+      .send(ratings);
+    expect(response.status).to.equal(400);
+    expect(response.body.message).to.equal('You cannot rate your article');
+    const numberOfRatings = await BaseRepository.findAll(Rating, {});
```

Comment:
use sequelize `count` when you **only** need the count.

Get the initial count before making the request. Then compare the counts before and after the request.